src/app/accounts/add-wallet.ts

```diff
@@ -0,0 +1,30 @@
+import { AccountsRepository, WalletsRepository } from "@services/mongoose"
+
+export const addWallet = async ({
```

Comment:
`addWallet` is no longer needed.
Since there is no update to Account (as setting `defaultWalletId` happens elsewhere) it collapses to:

```
export const addWallet = async (
{  accountId,
   type,
   currency,
 }: {
   accountId: AccountId
   type: WalletType
   currency: WalletCurrency
 }): Promise<Wallet | ApplicationError> => {
  return await WalletsRepository().persistNew({
     accountId,
     type,
     currency,
   })
}
  
   ```
   
   In other use cases `WalletRepository` should be called directly - there is no code saved by having this wrapper.
   The only time to introduce this trivial wrapper is to expose the use case to GQL (which isn't until later).

Reply:
there is an additional safety check on whether `accountId` is valid. I've also start build with it with this PR https://github.com/GaloyMoney/galoy/pull/935 so removing it to add it back in the next PR is not very relevant IMO